cmd/fyne_demo/main.go

```diff
@@ -84,8 +84,9 @@ func main() {
 
 	tabs := widget.NewTabContainer(
```

Comment:
I agree with Welcome as the first, and Advanced as the last. Windows (&Dialogs) towards the end also makes sense. Containers (&Layouts) are about surfacing to the users so being next to Windows makes sense.
I'm not sure about Widgets and Graphics. Graphics (canvas) are the building blocks so maybe that should be right after Welcome. Widgets build on Graphics so I'd put that next.

- Welcome
- Graphics
- Widgets
- Containers
- Windows
- Advanced

(just noticed it goes W, not W, W, not W, W, not W which is appealing for absolutely no reason :P)

Reply:
I had the same thought, but then I wasn't sure about what we should be presenting if people thought of this as order of importance. The canvas is really an enabling API rather than the main thrust for the library?

Reply:
Yes I see your point, we're not just making a graphics library, we're trying to make a library for creating beautiful UIs and widgets are the heart of that.
I saw the demo kind of like a tour, starting with welcome, basic building blocks, functional elements, positional elements, and so on. Maybe it should match tour.fyne.io, but I noticed that goes Introduction, Basics, Canvas, Layout, Widgets :/

Reply:
Well, I went with it - consistency with the tour is a good idea, perhaps some language tweaking on the other side will even that up